Before: e7ae54bc536b7c40370cf5434d7e73c725d096288b9a748340b993792debaac7
After: ecb07501b873ab031a49801b6b9f9237a53a2b3d9a70dec2d1bb3521975247ea
Patch: ab8eaee036439e06dc41565e1e64db2e68a1b1c6c87ad3ca3f03dff40776f12b
--- corvid-features/plugin.rb	2012-08-06 19:06:33.999477610 +1000
+++ /dev/null	2012-07-30 09:37:23.066384672 +1000
@@ -1,6 +0,0 @@
-def install
-  empty_directory 'resources'
-  empty_directory 'resources/latest'
-  copy_file       'resources/.gitignore'
-  install_feature 'test_spec', run_bundle: true, say_if_installed: false
-end
--- resources/.gitignore	2012-08-06 18:32:31.797559555 +1000
+++ /dev/null	2012-07-30 09:37:23.066384672 +1000
@@ -1 +0,0 @@
-latest
